docs/data/material/components/app-bar/app-bar.md

```diff
@@ -154,7 +154,7 @@ You can override this behavior by setting the `enableColorOnDark` prop to `true`
 
 {{"demo": "EnableColorOnDarkAppBar.js", "bg": true}}
 
-## Experimental APIs
+## Toolpad
 
 ### DashboardLayout
```

Comment:
Docs convention, I believe, no need to emphasize on a specific React module export here

```suggestion
### Dashboard Layout
```